modules/n1ql/pages/n1ql-language-reference/stringfun.adoc

```diff
@@ -1042,3 +1042,45 @@ SELECT UPPER("N1QL is awesome") as n1ql;
 }
 ----
 ====
+
+[#fn-str-upper]
+== URLENCODE(plain_string)
+
+=== Description
+Returns the input string encoded for use in a URL
```

Comment:
Full stop at end

```suggestion
Returns the input string encoded for use in a URL.
```

Reply:
I did wander about that full stop, I just copied what was there. Lesson for me — follow my intuition  💪